data/dialog phrases.txt

```diff
@@ -108,3 +108,23 @@ phrase "generic fleet bounty hunting on visit"
 phrase "generic pirate fleet battle on visit"
 	word
 		`You've landed on <planet>, but there are still pirates circling overhead. You should take off and help finish them off.`
+
+phrase "generic completed pirate cargo mission"
+	word
+		`You find the contact at the edge of the spaceport. Your cargo of <commodity> is removed from your ship and you collect your payment of <payment>.`
```

Comment:
```suggestion
		`You find the contact at the edge of the spaceport. Your cargo of <commodity> is removed from your ship, and you collect your payment of <payment>.`
```
Just need a comma here b/c two independent clauses.

Reply:
Yeah, this was a straight copy of the original material, but good call.